src/ColinHDev/Elevator/Elevator.php

```diff
@@ -18,26 +20,26 @@
 class Elevator extends PluginBase
 {
     /** @var MyPlot $myplot */
-    public $myplot;
+    public $myplot; //TODO: add support for CPlot ;)
```

Comment:
The APIs might be too different, especially after converting to libasynql and await-generator, to use this for both plugins.

Reply:
The check for MyPlot is only on startup. It can be hybrid.

Reply:
But the APIs are totally different, e.g. we don't even need the Main class of CPlot since nothing useful is in it